2020-InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -225,38 +228,38 @@ public void disabledPeriodic()
   public void initialize()
   {
     robotInitialized = true;
-    if(ImuEnable)
+    if(ImuEnable && robotInitialized != false)
```

Comment:
The purpose of the `robotInitialized != false` isn't clear - you've set it to `true` just above here.

Reply:
This has been changed